static/gsApp/components/ai/AiSetupDataConsent.tsx

```diff
@@ -50,6 +50,7 @@ function AiSetupDataConsent({groupId}: AiSetupDataConsentProps) {
     shouldShowBilling && organization.features.includes('seer-added');
 
   const isTouchCustomer = subscription?.type === BillingType.INVOICED;
+  const isSponsoredCustomer = subscription?.type === BillingType.PARTNER;
```

Comment:
we should check `subscription.isSponsored` instead since we only want to show that messaging for orgs on sponsored plans (not all partner orgs are on sponsored plans)

Reply:
Changed